bootenv/env.go

```diff
@@ -0,0 +1,56 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2023 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package bootenv
```

Comment:
the package itself probably needs a doc comment and maybe a clearer name, I'm not sure why it's called bootenv? it exists to offer key scope support for platforms not doing measured boot?

Reply:
I guess `bootenv` was the initial name because the key scope support is related to the boot environment (boot mode and models). Maybe a different name with the word "scope" in it? `bootscope`/`keyscope`? @chrisccoulson

Reply:
bootscope is not a bad name, adding a package doc comment is still also a good idea, as this is meant to be used by platforms that don't do measurements?

Reply:
I'm fine with bootscope as well.

Reply:
Renamed bootenv to bootscope and added package doc comment.